repos/system_upgrade/el7toel8/actors/grep/actor.py

```diff
@@ -0,0 +1,37 @@
+from leapp.actors import Actor
+from leapp.models import InstalledRedHatSignedRPM
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+from leapp.reporting import Report
+from leapp.libraries.common.reporting import report_with_remediation
+
+
+class Grep(Actor):
+    """
+    Check if Grep is installed. If yes, write information about non-compatible changes.
+    """
+
+    name = 'grep'
+    consumes = (InstalledRedHatSignedRPM,)
+    produces = (Report,)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        for fact in self.consume(InstalledRedHatSignedRPM):
+            for rpm in fact.items:
+                if rpm.name == 'grep':
+                    report_with_remediation(
+                        title='Grep incompatible changes in the next major version',
```

Comment:
Missed has? 'Grep has incompatible..'

Reply:
Fixed, IIRC I took it from some other actor from the repo - I thought it's the way how it's written there, so I wanted to stay consistent.